src/absil/ilread.fs

```diff
@@ -3601,11 +3602,11 @@ let openMetadataReader (fileName, mdfile: BinaryFile, metadataPhysLoc, peinfo, p
     let inbase = Filename.fileNameOfPath fileName + ": "
 
     // All the caches. The sizes are guesstimates for the rough sharing-density of the assembly 
-    let cacheAssemblyRef = mkCacheInt32 reduceMemoryUsage inbase "ILAssemblyRef" (getNumRows TableNames.AssemblyRef)
+    let cacheAssemblyRef = mkCacheInt32 false inbase "ILAssemblyRef" (getNumRows TableNames.AssemblyRef)
```

Comment:
Why not make everything "false" here?I feel like we should be caching everything.

Reply:
Becasause
1. These caches themselves take space 
2. It's not clear it's worth it - if the items aren't being duplicated in long term storage the cache isn't worth it - so we should work based off evidence that duplication is happening.